environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
   - nodefaults
 dependencies:
   - python=3.9
+  - flask-basicauth=0.2.0
```

Comment:
Oh wait, I'm not sure this is the right place 🤔  Might need to be within `setup.py` 🤔